src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+  const marriedMessage = `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`;
```

Comment:
It's difficult to read, create variable for this value`${sex === 'm' ? 'wife' : 'husband'}`